.evergreen/run-custom-csfle-tests.sh

```diff
@@ -103,8 +105,3 @@ if [ $DRIVER_CSFLE_TEST_RESULT -ne 0 ]; then
   echo "Driver tests failed, look above for results"
   exit 1
 fi
-
-echo "Test legacy version of FLE bindings"
-rm -rf node_modules/mongodb-client-encryption
-npm install mongodb-client-encryption@"^1.2.7"
-npm run check:csfle
```

Comment:
We might need to discuss EOLing that version if we're going to stop integration testing it.

Reply:
Chatted in standup: We will put up a deprecation notice on npm after we release the official 2.0 version and we can indeed remove this testing.

Reply:
As per discussion will EOL when 2.0.0 fle is released.